mscore/pagesettings.cpp

```diff
@@ -81,9 +88,46 @@ PageSettings::PageSettings(QWidget* parent)
 
 PageSettings::~PageSettings()
       {
+      QSettings settings;
+
```

Comment:
@Jojo-Schmitz  I think this is the problem you are talking about,  "spaces" (which is disturbing the coding style and consistency).